### ISSUE TITLE
Virtual Methods

### ISSUE THREAD

**Issue by JasonHJX** [yuyang9119/UnityAssetStoreSupport#31]
Does TweenPlayer include some static Virtual Methods? I can call and implement simple animations directly in the program without add the mono component. Like this:
<img width="500" alt="visual method" src="https://github.com/user-attachments/assets/fa23b33d-b839-4840-b07c-f3f34a8ef862">

**Comment by YT-GameMaker** [comment]
Yes, TweenPlayer includes some static methods that correspond to the curves displayed in the Inspector. You can use these interpolation methods with the formula: 
`(to - from) * Interpolator.MethodX(t) + from`
